app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User
```

Comment:
Please use devise for the user and session management. See the user model in Casematch as an example: https://github.com/eonum/casematch/blob/master/app/models/user.rb